try {
    console.log("hii");
    console.log(x);
    let x =  12;

    console.log("hii 2");
} catch (error) {
    console.log(error);
}

console.log("hii 3");
